src/redux/reducers/userReducer.ts

```diff
@@ -106,5 +109,35 @@ export const userReducer = createReducer(initialState, {
     GET_USERS_FAILED: (state, action) => {
         state.isLoading = false;
         state.users = action.payload;
-    }
+    },
+    LOAD_TARGETS_REQUEST: (state, action) => {
+        state.targets = [];
+    },
+    LOAD_TARGETS_SUCCESS: (state, action) => {
+        console.log(action.payload)
+        state.targets = action.payload.targets;
+    },
+    LOAD_TARGETS_FAILED: (state, action) => {
+        state.targets = []
+    },
+    CREATE_TARGET_REQUEST: (state, action) => {
+
+    },
+    CREATE_TARGET_SUCCESS: (state, action) => {
+        console.log({subtitle: action.payload.subtitle, subtargets: action.payload.subtargets})
+        state.targets = [...state.targets, {name: action.payload.subtitle, subTargets: action.payload.subtargets}]
```

Comment:
If we're working with our reducers in a mutable way, we can use `push` method for readability